src/utils/mtev_logic.c

```diff
@@ -505,14 +543,19 @@ mtev_logic_exec_predicate(mtev_logic_exec_t *exec, mtev_logic_pred_t *pred, void
     }
   } else {
     if(lhs.type == MTEV_LOGIC_STRING) {
+      char ntcopy[128];
```

Comment:
magic number from where?

Reply:
From me. I chose it because it seems big enough.